### PR TITLE
Add test annotation to annotations section

### PR DESCRIPTION
**Pull request by gautamjajoo**
The Upload Test Annotations is moved from the `edit phase modal` to the `Test and Annotations` section.
@Ram81 @Kajol-Kumari

Screenshot from 2021-06-28 22-20-03

### CODE REVIEW

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@gautamjajoo please do the following -
1. After test annotation is finished let's clear the selected phase value.
2. The text `Test Annotation` and dropdown is currently on the same row, let make it similar to `Edit phase details` i.e keep the heading `Test Annotation:` in one row and the dropdown in the next row a bit right aligned.

File: frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts [hunk 1]
```diff
@@ -557,6 +557,55 @@ export class ChallengesettingsComponent implements OnInit, OnDestroy {
     SELF.globalService.showModal(PARAMS);
   }
 
+  /**
+   * Edit test annotation of the phase
+   */
+  editTestAnnotation() {
```
[the remark this answers is not in the document]
Comment (gautamjajoo) [reply frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.ts:563]: Actually the service for updating the phase name is added in the phase PR, so I will wait it to be merged, then I will update this PR!

**Comment by Ram81** [comment]
@gautamjajoo can we make the options in this section look better with appropriate padding. The section looks quite bad right now. Either move all options in a separate row or add required padding between rows and columns

**Comment by gautamjajoo** [comment]
@Ram81 Actually, when I will be making the tab view these things would also change, that's why I have left that part.

**Comment by Ram81** [comment]
@gautamjajoo we still have to fix the  style for this section if we want to merge it. The current style looks quite bad. Otherwise, you can open a new PR after tab view is merged.

**Comment by gautamjajoo** [comment]
Okay, so we can add this and the note for test annotation part after tab view is merged.

**Comment by gautamjajoo** [comment]
@Ram81 @Kajol-Kumari How does this look? Can you suggest changes, if needed?
Screenshot from 2021-07-17 17-31-18

**Comment by Kajol-Kumari** [comment]
> @Ram81 @Kajol-Kumari How does this look? Can you suggest changes, if needed?
> Screenshot from 2021-07-17 17-31-18

@gautamjajoo How about making the UI somewhat similar to ⬇️ 
Screenshot 2021-07-17 at 9 28 59 PM

**Comment by gautamjajoo** [comment]
By mentioning `Annotation` as heading, I thought it would be easier for hosts to know the reason behind the selection of phase in this section. I can change it to any of the above, as you say.
@Ram81 @Kajol-Kumari

**Comment by Ram81** [comment]
@gautamjajoo it should just be 1 heading `Evaluation and Annotations`.  And we should move the test annotations section in `Edit phase details` section now. As we already have phase dropdown added there. We can put it back in the modal that pops up when we try to edit the phase details. It would be bad UX to have multiple dropdowns for phase details edit and annotations edit. Sorry about the back and forth.

**Comment by Ram81** [comment]
In that tab view of settings page we will put `Evaluation script and guidelines` under evaluation tab with worker logs and worker status UI. We won't need a separate section for annotations update it can be part of edit phase details

**Comment by gautamjajoo** [comment]
So, the test annotation part would be in the edit phase detail modal, right?
So where would the note be when the file size > 100mb?

**Comment by gautamjajoo** [comment]
Closing the PR since we don't need this feature now.

**Pull request closed without merging**